dev-tools/ls-apis/src/api_metadata.rs

```diff
@@ -199,6 +199,14 @@ pub struct ApiMetadata {
     pub server_package_name: ServerPackageName,
     /// human-readable notes about this API
     pub notes: Option<String>,
+    /// whether this package is ever expected to be deployed in a real system
+    dev_only: Option<bool>,
```

Comment:
What does None mean?

Reply:
Updated the comment to reflect that `None` means an API *is* deployed on real systems.

Reply:
Ah thanks. Worth using serde's default for this?

Reply:
We definitely could.  I don't have a strong feeling either way.